server/src/utils/chess-game-util.test.js

```diff
@@ -0,0 +1,23 @@
+import CGUtil from './chess-game-util';
```

Comment:
Please add a test that returns actual numbers that you expect to get when you send some value.
like sending (winner=p1, p1=1400, p2=1400) returns (p1=1450, p2=1350).